module1/projects/connect_four/iteration_2.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+title: Iteration 2 - Placing Pieces
+---
+
+_[Back to Connect Four Home](./index)_
+_[Back to Requirements](./requirements)_
+
+## Test Driven Development
+
+In this iteration, you are required to use TDD to create your classes. Use the interaction pattern to determine what a method should do and write one or more tests to verify that expected behavior. Then you can implement the method. You should always write code with the purpose of making a test pass.
+
+## Placing Pieces
+
+Update your program to request user input and allow them to place an individual piece.
+
+Your program should ask for the user to enter a letter A - G, update the board so that their piece has been placed in that column, and then display the board again.
+
+Your program should ask the user to enter a letter A-G.
+Update the board to display the player's piece in the lowest available position of the selected column with an 'X'.
```

Comment:
Lines 17-20 repeat the same information... I think lines 19-20 are probably the most descriptive and should be kept.

Reply:
Fixed, thanks for catching that!